src/features/annotation-thread/AnnotationThread.scss

```diff
@@ -0,0 +1,10 @@
+@import 'src/styles/variables';
+
+$annotationThread-width: 340px;
```

Comment:
I believe we're following SUIT style naming for our sass variables, should this be `$AnnotationThread-width`?